engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1965,42 +1965,6 @@ private IdentifiedLocation getIdentifiedLocation(Tree ast, int b, int e, Option<
     };
   }
 
-  private IdentifiedLocation getIdentifiedLocation(ArgumentDefinition ast) {
-    // Note: In the IR, `DefinitionArgument` is a type of AST node; in the parser, it is not an AST-level type, but a
-    // type used only in specific locations. As a result, the IR expects it to have a source-code location, but the
-    // parser doesn't have a span for it. Here we synthesize one. This will not be necessary if we refactor IR so that
-    // `ArgumentDefinition` is not an AST node, though that change would have effects throughout the compiler and may
-    // not be worthwhile if IR is expected to be replaced.
-    long begin;
-    if (ast.getOpen() != null) {
-      begin = ast.getOpen().getStartCode();
-    } else if (ast.getOpen2() != null) {
```

Comment:
- it is great the removal of this method is possible!
- I didn't like it to begin with https://github.com/enso-org/enso/pull/3716/files#r986401625
- Maybe there were reasons to have it in the past: https://github.com/enso-org/enso/pull/3716/files#r986444388
- but times has changed since #3716
- and it is great the simple `getIdentifiedLocation(def.getPattern())` is good enough!